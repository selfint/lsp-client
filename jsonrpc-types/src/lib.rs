mod notification;
mod request;
mod response;

pub use notification::Notification;
pub use request::Request;
pub use response::{JsonRPCError, JsonRPCResult, Response};

pub(crate) const JSONRPC_V2: &str = "2.0";

#[cfg(test)]
/// Helpers for serialization/deserialization tests
pub(crate) mod tests;
