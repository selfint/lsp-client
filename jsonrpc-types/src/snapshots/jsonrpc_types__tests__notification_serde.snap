---
source: jsonrpc-types/src/lib.rs
expression: "Notification::new(\"method\", ())"
---
{
  "jsonrpc": "2.0",
  "method": "method",
  "params": null
}
